docs/src/main/sphinx/udf/python.md

```diff
@@ -0,0 +1,185 @@
+# Python user-defined functions
+
+A Python user-defined function is a [user-defined function](/udf) that uses the
+[Python programming language and statements](python-udf-lang) for the definition
+of the function.
+
+:::{warning}
+Python user-defined functions are an experimental feature.
+:::
+
+## Python UDF declaration
+
+Declare a Python UDF as [inline](udf-inline) or [catalog UDF](udf-catalog) with
+the following steps:
+
+* Use the [](/udf/function) keyword to declare the UDF name and parameters.
+* Add the `RETURNS` declaration to specify the data type of the result.
+* Set the `LANGUAGE` to `PYTHON`.
+* Declare the name of the Python function to call with the `handler` property in
+  the `WITH` block.
+* Use `$$` to enclose the Python code after the `AS` keyword.
+* Add the function from the handler property and ensure it returns the declared
+  data type.
+* Expand your Python code section to implement the function using the available
+  [Python language](python-udf-lang).
+
+The following snippet shows pseudo-code:
+
+```text
+  FUNCTION python_udf_name(input_parameter data_type)
+    RETURNS result_data_type
+    LANGUAGE PYTHON
+    WITH (handler = 'python_function')
+    AS
+    $$
+    ...
+    def python_function(input):
+        return ...
+    ...
+    $$
+```
+
+A minimal example declares the UDF `doubleup` that returns the input integer
+value `x` multiplied by two. The example shows declaration as [](udf-inline) and
+invocation with the value `21` to yield the result `42`.
+
+Set the language to `PYTHON` to override the default `SQL` for [](/udf/sql).
+The Python code is enclosed with ``$$` and must use valid formatting.
+
+```text
+WITH
+  FUNCTION doubleup(x integer)
+    RETURNS integer
+    LANGUAGE PYTHON
+    WITH (handler = 'malzwei')
```

Comment:
Is this a made up word? Can we choose something more understandable here?

Reply:
it means timestwo in german .. I will change ;-)